client/src/context/DataProvider.tsx

```diff
@@ -292,6 +292,7 @@ export const DataProvider = ({ children }: { children: JSX.Element }) => {
                     events: [],
                     tasks: tasks.map((t: ITask) => t.id),
                     creationTime: data.creationTime,
+                    isPublic: false
```

Comment:
should read this from data and not just set to be false, need to change `ProjectGet` in project.go handler to return ispublic status